src/frontend/src/lib/types/token.ts

```diff
@@ -40,7 +40,7 @@ const TokenBuyableSchema = z.object({
 	buy: z.custom<AtLeastOne<TokenBuy>>().optional()
 });
 
-const TokenSchema = z
+export const TokenSchema = z
```

Comment:
I did not provided a test for this. I guess we should move it to a `token.validation` in another PR and add a test. I can do it before this PR if you wish?

Reply:
https://github.com/dfinity/oisy-wallet/pull/3340